tests/coffee_script/CoffeeLintBearTest.py

```diff
@@ -11,6 +11,228 @@ class Gangster
 """.splitlines(keepends=True)
 
 
+good_file_use_english_operator = """
+1 or 1
+1 and 1
+1 isnt 1
+1 is 1
+""".splitlines(keepends=True)
+
+
+bad_file_use_english_operator = """
```

Comment:
Ah nvm. So I guess this is the trend `good_file_<property-check>` and `bad_file_<property-check>`